app/layout/community/published-projects/detail/share/component.tsx

```diff
@@ -54,15 +54,15 @@ export const ProjectShare: React.FC<ProjectShareProps> = () => {
 
   return (
     <div>
-      <h3 className="mb-6 text-sm font-semibold text-white">Share</h3>
+      <h3 className="mb-6 text-sm font-semibold">Share</h3>
 
       <div className="space-y-5">
         <button
```

Comment:
![image](https://user-images.githubusercontent.com/8928965/128081452-c3dd9d86-31e6-4819-bd69-fd4b24710b60.png)

Can you add some hover for these links? Underline them when you hover